signing_service/src/signing_service.py

```diff
@@ -126,6 +127,40 @@ def _validate_arguments(self):
 
 
 def _parse_arguments() -> argparse.Namespace:
+
+    def get_secret_variable_wrapper(env_variable_name=None, file=False, base64_convert=False):
+        def wrapper(**kwargs):
+            return GetSecretVariable(env_variable_name, file, base64_convert, **kwargs)
+
+        return wrapper
+
+    class GetSecretVariable(argparse.Action):
+        def __init__(self, env_variable_name, file, base64_convert, option_strings, dest, nargs=None):
+            self.env_variable_name = env_variable_name
+            self.file = file
+            self.base64_convert = base64_convert
+            if self.env_variable_name:
+                nargs = 0
```

Comment:
But it is necessary in this case because if we call `--ethereum-private-key-from-env` we don't need any parameter for this argument and exactly `nargs` is responsible for this